src/components/pages/HomePage/index.jsx

```diff
@@ -1,3 +1,4 @@
+//#region > Imports
 //> React
 // Contains all the functionality necessary to define React components
 import React from "react";
```

Comment:
Remove newline below this line. (Cannot be marked)

Reply:
We split imports per header and a new line.